examples/src/utils/isUrlPointCloudModel.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Returns true if the URL provided points to a point cloud model.
+ * @param addModelOptions 
+ * @returns 
+ */
+export async function isUrlPointCloudModel(modelBaseUrl: string) {
```

Comment:
Could we name this `isLocalUrlPointCloudModel`? Just to ensure no one tries using it for remote requests that e.g. may require authentication.